packages/cli/src/ceramic-cli-utils.ts

```diff
@@ -101,9 +101,9 @@ export class CeramicCliUtils {
 
     let configFromCli: DaemonConfig
     {
-      let _corsAllowedOrigins: string | RegExp[] = '*'
-      if (corsAllowedOrigins != null && corsAllowedOrigins != '*') {
-        _corsAllowedOrigins = corsAllowedOrigins.split(' ').map((origin) => new RegExp(origin))
+      let _corsAllowedOrigins: string[] = ['*']
```

Comment:
This is the change I'm most worried about - I don't want to accidentally break the common case where all origins should be allowed.  I'm also not really sure how to test these changes.

Reply:
@stbrody this works so we can use RegExp[]
```
a = new RegExp('^\\*$')
> /^\*$/
a.test('*')
> true
```

Reply:
Oh actually I take that back I guess the default here should be to match anything

`.*`

Reply:
ah yes, of course, thanks.

Updated to use array of RegExp as the type